### PR TITLE
daide string generation and tests

### LINKED ISSUE #26: Support DAIDE string construction from keyword tokens
I am from SHADE and would like to use this library as our main parser. We are currently using this library: https://github.com/trigaten/DAIDE, which provides simple parsing, as well as DAIDE message construction using DAIDE keywords, like the following:

```python
# easily compose DAIDE keywords
>>> from DAIDE import Order, Unit, ORR, HLD
>>> arrangement = ORR([Order(Unit("FFF FFF FFF"), HLD()), Order(Unit("FFF FFF FFF"), HLD())])
>>> str(arrangement)
'ORR ((FFF FFF FFF) HLD) ((FFF FFF FFF) HLD)'
```

It would be very useful if your parser provided this support. How difficult would this be to add?

### PR DESCRIPTION
**Pull request by byung-o**
resolves #26

### CODE REVIEW

File: src/daidepp/constants.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
+from typing import Literal
```
Comment (mjspeck) [inline src/daidepp/constants.py:1]: This doesn't work in Python 3.7. You need to use `from typing_extensions import Literal`

File: src/daidepp/keywords.py [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+
+@dataclass
+class PCE:
+    powers: List[POWER]
+
+    def __init__(self, *powers):
+        self.powers = powers
+
+    def __str__(self):
+        return "PCE (" + " ".join(self.powers) + ")"
```
Comment (mjspeck) [inline src/daidepp/keywords.py:136]: I think all parentheses should have a whitespace character separating them from the following token.

**Review by mjspeck — CHANGES_REQUESTED** [review]
I got a couple of formatting changes when I ran `black` so you might need to run it on your end. If you did the changes could be due to a difference in configurations which we should address in another ticket. Aside from that, there are a couple of minor changes to make which I've listed in the comments. Once those are addressed this should be good to go.

File: src/daidepp/keywords.py (hunk 2 above)
Comment (byung-o) [reply src/daidepp/keywords.py:136]: So you're saying instead of something like `PRP (PCE (AUS ENG))`, it should be `PRP ( PCE ( AUS ENG ))` ?

Comment (mjspeck) [reply src/daidepp/keywords.py:136]: Yup `PRP ( PCE ( AUS ENG ) )`. That's how all Albert bot messages are formatted.

**Review by mjspeck — APPROVED** [review]
LGTM!

**Pull request merged**